src/hydra-queue-runner/state.hh

```diff
@@ -123,6 +123,7 @@ struct Build
     BuildID id;
     nix::StorePath drvPath;
     std::map<std::string, nix::StorePath> outputs;
+    int jobsetId;
```

Comment:
`int` is probably an odd type here. `BuildID` is an `unsigned int`, might be better to use that here as well, maybe even duplicate the typedef for `JobsetID` should we ever have to switch that to long.

Reply:
Worth noting that the builds table has hundreds of millions of rows and the jobsets table has ~20, though that is definitely not universally true.